base/applications/notepad/text.c

```diff
@@ -83,21 +110,25 @@ ReadText(HANDLE hFile, LPWSTR *ppszText, DWORD *pdwTextLen, ENCODING *pencFile,
     pBytes[dwSize + 1] = '\0';
 
     /* Look for Byte Order Marks */
-    if ((dwSize >= 2) && (pBytes[0] == 0xFF) && (pBytes[1] == 0xFE))
+    if ((dwSize >= 2) && memcmp(pBytes, "\xFF\xFE", 2) == 0)
```

Comment:
Not sure this is more efficient. The original code avoided an unneeded function call.

Reply:
Reverted.